best-practices/README.md

```diff
@@ -109,7 +109,7 @@ Testing
 * Avoid `any_instance` in rspec-mocks and mocha. Prefer [dependency injection].
 * Avoid `its`, `let`, `let!`, `specify`, and `before` in RSpec.
 * Avoid using `subject` explicitly *inside of an* RSpec `it` block.
-  [Example][subject-example].
```

Comment:
Good catch. That "explicit subject example" also references an out of date file. Updating it but using that style.